libraries/core-react/src/components/Menu/Menu.tsx

```diff
@@ -100,6 +100,7 @@ export const Menu = React.forwardRef<HTMLUListElement, MenuProps>(function Menu(
     containerRef.current,
     null,
     placement,
+    0,
```

Comment:
`0` looks very "tight" did you check with @equinor/eds-ux that this was ok value? I think value was more like 2 or 4px before... 🤔

Reply:
I used the context menu example [here](https://www.figma.com/file/luNobpDIWTuQ9rhyovGLBI/Desktop?node-id=977%3A27105) and it's 0px from anchor to menu

Reply:
4px 
![image](https://user-images.githubusercontent.com/1070981/114409000-48058900-9baa-11eb-8505-579a2ea69604.png)

vs 

2px 
![image](https://user-images.githubusercontent.com/1070981/114409255-8b5ff780-9baa-11eb-8f0d-c65b2f37f0ca.png)

vs 

0px 
![image](https://user-images.githubusercontent.com/1070981/114409165-71261980-9baa-11eb-8736-d7d09f8940e7.png)

Reply:
I would say maybe `2px`? What do you guys think @equinor/eds-ux ?

Reply:
@mimarz @pomfrida I think it shall be 4px space between the button and the drop menu. I guess the clickbound take care of that?